src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -338,58 +343,90 @@ class GridEditable<
     grid.style.gridTemplateColumns = columnWidths.join(' ');
   }
 
-  renderHeaderButton = () => {
-    if (!this.props.isEditable) {
-      return null;
-    }
+  renderHeaderButtons() {
+    const {noEditMessage, editFeatures} = this.props;
+    const renderDisabled = p => (
+      <Hovercard
+        body={
+          <FeatureDisabled
+            features={p.features}
+            hideHelpToggle
+            message={noEditMessage}
+            featureName={noEditMessage}
```

Comment:
Shouldn't this just be "Discover Query" or something?

Reply:
The hovercard for sentry standalone ends up looking like this with the current text.

![Screen Shot 2020-01-24 at 4 38 33 PM](https://user-images.githubusercontent.com/24086/73105894-1da5ae00-3ec8-11ea-86db-9a66c4af75ba.png)